packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -203,6 +236,48 @@ class PageTransitionSwitcher extends StatefulWidget {
   /// The child provided to the transitionBuilder may be null.
   final PageTransitionSwitcherTransitionBuilder transitionBuilder;
 
+  /// A function that wraps all of the children that are transitioning out, and
+  /// the [child] that's transitioning in, with a widget that lays all of them
+  /// out. This is called every time this widget is built. The function must not
+  /// return null.
+  ///
+  /// The default [PageTransitionSwitcherLayoutBuilder] used is
+  /// [defaultLayoutBuilder].
+  ///
+  /// The following example shows a [layoutBuilder] that places all entries inside
+  /// a [Column] that sizes its height depending on the heights of active entries.
+  ///
+  /// ```dart
+  /// PageTransitionSwitcher(
+  ///   duration: const Duration(milliseconds: 100),
+  ///   child: Container(color: Colors.red),
+  ///   layoutBuilder: (
+  ///     List<Widget> entries,
+  ///   ) => Column(
+  ///     children: entries
+  ///   ),
```

Comment:
nit: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods
```suggestion
  ///   ) {
  ///     return Column(children: entries);
  ///   },
```